templates/slides.tmpl

```diff
@@ -105,8 +105,25 @@
         languages: ['bash', 'markdown', 'go', 'java']
       })
 
+      var reLanguageTag = /^\s*language:(\w+)\n?([\s\S]*)/m
```

Comment:
lazy.

should probably be a substring to separate first line from the rest, then regex only has to match tag a short string

would make the following monstrosity a little less blergh too